### PR TITLE
Font and text adjustments. Adding survey

### PR DESCRIPTION
**Pull request by jardakotesovec**

### CODE REVIEW

File: templates/pfl.tpl [hunk 1]
```diff
@@ -216,8 +223,32 @@
             </div>
 
         </div>
-
     </div>
-
+        <div id="ff-compose"></div>
+        <script>
+            window.journalISSN="testingISSN"
```
Comment (asmecher) [inline templates/pfl.tpl:229]: You should be able to use, off the top of my head:
```
window.journalISSN="{$currentJournal->getData('onlineIssn')|default:$currentJournal->getData('printIssn'):default:'unknown'|escape}"
```
...but see my other comment on escaping.

File: templates/pfl.tpl [hunk 2]
```diff
@@ -216,8 +223,32 @@
...
+            window.pflRecordSurveySubmit = function(data) {
+                try {
```
Comment (asmecher) [inline templates/pfl.tpl:231]: Smarty uses `{` `}` for its own syntax, so it's generally necessary to wrap any Javascript in a Smarty template inside `{literal}` / `{/literal}` tags. See e.g.: https://github.com/asmecher/lensGalley/blob/main/templates/display.tpl#L13..L28

**Comment by asmecher** [comment]
@jardakotesovec, I cherry-picked the first 2 commits from this PR into `main` already, but added a couple of comments to the last one re: the survey:

https://github.com/asmecher/pflPlugin/pull/14/commits/e9a1c85ed51e9e7b7a5d44e73922ea2cccc963e8

File: templates/pfl.tpl (hunk 1 above)
Comment (jardakotesovec) [reply templates/pfl.tpl:229]: Indeed works.. so I included it..

File: templates/pfl.tpl (hunk 2 above)
Comment (jardakotesovec) [reply templates/pfl.tpl:231]: There already has been JS.. and as long as there is spacing around curly braces its fine. (https://www.smarty.net/docs/en/language.escaping.tpl).

**Comment by jardakotesovec** [comment]
Closing this one as I create two new separate ones with additions.

**Pull request closed without merging**